web/src/installers/index.ts

```diff
@@ -594,10 +594,7 @@ export class Installer {
       "19.03.4",
       "18.09.8",
     ],
-    containerd: [
-      "1.3.7",
-      "1.2.13",
-    ],
+    containerd: [ "1.3.7", "1.2.13"], // cron-containerd-update
```

Comment:
why no 1.4.3 here?

Reply:
And 1.3.9

Reply:
I was leaving those for the automated PR to fill in to confirm it's working.